test/components/article.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Article from '../../src/components/article';
+import { createRemarkPage, createPageInfo } from '../__fixtures__/page';
+
+const ShallowRenderer = require('react-test-renderer/shallow');
```

Comment:
Does an ES6 import not work here?

```suggestion
import ShallowRenderer from 'react-test-renderer/shallow';
```

Reply:
Thanks for looking at this. I was getting some TypeScript errors but should be fixed now 🎉